proposals/ref-unsafe-in-iterators-async.md

```diff
@@ -155,6 +162,9 @@ class C
   > or the variable needs to be hoisted as part of an async ([§15.15][async-funcs]) or an iterator ([§15.14][iterators]) method**.
   > - [...]
 
+  - Currently, we have an existing C# 12 warning for address-of in async methods and a proposed C# 13 error for address-of in iterators.
```

Comment:
>a proposed C# 13 error for address-of in iterators [](http://example.com/codeflow?start=85&length=50)

What does "a proposed C# 13 error for address-of in iterators" mean? Is it a new error reported by new compiler, or it is an error reported for language version C# 13 and later? Consider clarifying.

Reply:
It doesn't really make a difference if it were reported for earlier langversions since address-of in iterators could not be used anyway (because unsafe code in general could not be used inside iterators). But will clarify, thanks.